src/components/ClusterManager.js

```diff
@@ -318,7 +318,7 @@ export default class ClusterManager extends PureComponent {
           div({ style: { marginLeft: '0.5rem', paddingRight: '0.5rem', color: colors.dark() } }, [
             div({ style: { fontSize: 12, fontWeight: 'bold' } }, 'Notebook Runtime'),
             div({ style: { fontSize: 10 } }, [
-              span({ style: { textTransform: 'uppercase', fontWeight: 500 } }, currentStatus || 'None'),
+              span({ style: { textTransform: 'uppercase', fontWeight: 500 } }, [currentStatus === 'Reconfiguring' ? 'Updating' : (currentStatus || 'None')]),
```

Comment:
This feels a little weird to be transforming the text in only one case, but I do agree that `Updating` is more clear than `Reconfiguring`. Is `Reconfiguring` how the API returns it?